### DISCUSSION TITLE
WireMock-Captain at Open Source 101!

### DISCUSSION THREAD

**Discussion by satvik-s** [HBOCodeLabs/wiremock-captain#75] [Announcements]
We will be doing a session at [Open Source 101](https://opensource101.com/) talking about WireMock-Captain at 1:30p ET on March 29!

**Comment by satvik-s** [comment]
YouTube upload for the session: https://youtu.be/K0atgOS_Yj0

**Reply by OfficialAlius** [reply]
I need friends
